internal/cluster/validator.go

```diff
@@ -534,8 +536,13 @@ func (v *clusterValidator) networkPrefixValid(c *clusterPreprocessContext) (Vali
 		return ValidationPending, "The Cluster Network CIDR is undefined."
 	}
 	validClusterNetworks := funk.Filter(c.cluster.ClusterNetworks, func(clusterNetwork *models.ClusterNetwork) bool {
-		return clusterNetwork != nil &&
-			network.VerifyNetworkHostPrefix(clusterNetwork.HostPrefix) == nil &&
+		if clusterNetwork == nil {
+			return false
+		}
+		if v.skipNetworkHostPrefixCheck(c, clusterNetwork.HostPrefix) {
```

Comment:
I actually thought to set it before  validClusterNetworks , i mean there is no point to validate anything if you know you should skip it no?

Reply:
The thought was to skip the validation *only* if the prefix was not set and the networkTypes was not OVN of SDN.
If we take the approach that the validation should be skipped regardless of the prefix then we can move this check before.

Reply:
Updated to skip validations only based on networkType and ignore the prefix.